File: Assignments/Assignment_9/program1.c
// Write a program which accept number from user and display below pattern.

#include<stdio.h>

////////////////////////////////////////////////////////////////////
//
//  Function Name : Display
//  Description :   It is used to display pattern
//  Input :         Integer
//  Output :        pattern
//  Author :        Omkar Sachin Narlawar
//  Date :          26/10/2025
//
////////////////////////////////////////////////////////////////////

void Display(int iNo)
{

    //Updater
    if(iNo < 0)
    {
        iNo = -iNo;
    }

    int iCnt = 0;

    for(iCnt = 1; iCnt <= iNo; iCnt++)
    {
        printf("*\t",iCnt);
    }

    for(iCnt = 1; iCnt <= iNo; iCnt++)
    {
        printf("#\t",iCnt);
    }
}

int main()
{
    int iValue = 0;

    printf("Enter number\n");
    scanf("%d",&iValue);

    Display(iValue);

    return 0;
}

File: Assignments/Assignment_26/program1.java
// Accept number from user and display below pattern.
// Input : 5
// OUTPUT : A   B   C   D   E

import java.util.*;

class Pattern
{
    public void Display(int iNo)
    {
        int iCnt = 0;
        char Alpha = '\0';

        Alpha = 'A';

        for(iCnt = 1; iCnt <= iNo; iCnt++)
        {
            System.out.print(Alpha+"\t");
            Alpha++;
        }
    }
}

class program1
{
    public static void main(String A[])
    {
        int iValue = 0;

        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter number of elements : ");
        iValue = sobj.nextInt();

        Pattern pobj = new Pattern();
        pobj.Display(iValue);
    }
}